packages/fx-core/src/component/feature/sso.ts

```diff
@@ -253,16 +253,12 @@ function getUpdateComponents(
   let needsBot = false;
   let needsTab = false;
   const teamsBotComponent = getComponent(projectSetting, ComponentNames.TeamsBot);
-  if (teamsBotComponent && !teamsBotComponent.sso) {
-    if (
-      teamsBotComponent.capabilities &&
-      teamsBotComponent.capabilities.length === 1 &&
-      teamsBotComponent.capabilities.includes("message-extension")
-    ) {
-      needsBot = false;
-    } else {
-      needsBot = teamsBotComponent.hosting !== ComponentNames.Function;
-    }
+  if (
+    teamsBotComponent &&
+    !teamsBotComponent.sso &&
+    teamsBotComponent.hosting !== ComponentNames.Function
```

Comment:
Includes command-response bot and restify notification bot?

Reply:
It keeps the same sso logic. Just remove message extension limitation

Reply:
Confirmed with @SLdragon, command-response bot has sso and restify notification bot does not have sso.